src/lib/autocomplete/autocomplete.ts

```diff
@@ -50,6 +58,9 @@ export class MdAutocomplete implements AfterContentInit {
   /** Function that maps an option's control value to its display value in the trigger. */
   @Input() displayWith: (value: any) => string;
 
+  /** Event that is emitted whenever an option from the list is selected. */
+  @Output() select: EventEmitter<MdAutocompleteSelect> = new EventEmitter<MdAutocompleteSelect>();
```

Comment:
`@Output` can be basically any `Observable`. `EventEmitter` is just like a RxJS `Subscription`. We originally changed all `@Output`s to just expose the `Observable`. See https://github.com/angular/material2/issues/440